modules/react/layout/spec/SSR.spec.tsx

```diff
@@ -12,9 +12,9 @@ describe('Layout Components', () => {
       expect(ssrRender).not.toThrow();
     });
   });
-  context('Stack Component', () => {
+  context('Grid Component', () => {
     it('should render on a server without crashing', () => {
-      const ssrRender = () => renderToString(<Stack spacing="s">Hello, Stack!</Stack>);
+      const ssrRender = () => renderToString(<Grid gridGap="s">Hello, Grid!</Grid>);
```

Comment:
Should this be `Flex` ?I might be missing some context!

Reply:
There is a `Flex` component option right above the `Grid` option